pkg/provider/handlers/deploy.go

```diff
@@ -247,7 +247,7 @@ func prepareEnv(envProcess string, reqEnvVars map[string]string) []string {
 }
 
 func getMounts() []specs.Mount {
-	wd, _ := os.Getwd()
+	wd := "/var/lib/faasd"
```

Comment:
It could be this:

`../faasd`

Or it could be an env-var with the actual path rather than putting it into code as above.

Reply:
(This would only be for certain mounts)